gavel/controllers/judge.py

```diff
@@ -86,6 +86,13 @@ def tx():
             elif request.form['action'] == 'Absent':
                 annotator.next.absent.append(annotator)
                 annotator.ignore.append(annotator.next)
+                if count >= settings.THRESHOLD_ABSENT:      #from here
+                    item_id = request.form['item_id']
+                    target_state = action == 'Disable'
+                    def tx():
+                        Item.by_id(item_id).active = False
+                        db.session.commit()                 #to here
```

Comment:
Please get rid of the `#to here` fluff comment. Same thing with `#from here` on line 89.